lib/FusionInventory/Agent/Inventory.pm

```diff
@@ -44,7 +44,8 @@ my %fields = (
                              SERIAL STEPPING FAMILYNAME FAMILYNUMBER MODEL
                              SPEED ID EXTERNAL_CLOCK ARCH CORECOUNT/ ],
     DRIVES           => [ qw/CREATEDATE DESCRIPTION FREE FILESYSTEM LABEL
-                             LETTER SERIAL SYSTEMDRIVE TOTAL TYPE VOLUMN/ ],
+                             LETTER SERIAL SYSTEMDRIVE TOTAL TYPE VOLUMN
+                             ENCRYPTION ENCRYPTALGO ENCRYPTED ENCRYPTTYPE/ ],
```

Comment:
ENCRYPTTYPE should be used as least by BSD support (waiting for @ddurieux method to rely encrypted fs to mount point).
About ENCRYPTED, from the api, win32 can report it as "Unknown", and maybe we can decide in some case to set it to "Partial", but honestly I still don't know if this could really happen.
Regarding the prefix, I understand your point of you and I would better expect this comment in the [specs PR](https://github.com/fusioninventory/fusioninventory.github.io/pull/60). If we follow your advice, maybe we can replace ENCRYPTED by ENCRYPT_STATUS and ENCRYPTION by ENCRYPT_MODEL. @ddurieux @orthagh any though ?